gemrb/GUIScripts/bg1/Start.py

```diff
@@ -52,45 +52,18 @@ def OnLoad():
 def SinglePlayerPress():
 	
 	SinglePlayerButton.SetText(13728)
+	MultiPlayerButton.SetStatus(IE_GUI_BUTTON_ENABLED)
 	MultiPlayerButton.SetText(13729)
 	ExitButton.SetText(15416)
 	MultiPlayerButton.OnPress (LoadSingle)
 	SinglePlayerButton.OnPress (NewSingle)
 	ExitButton.OnPress (BackToMain)
 	ExitButton.MakeEscape()
+	MoviesButton.SetText("")
 	if HasTOTSC():
 		MoviesButton.OnPress (MissionPack)
 		MoviesButton.SetText(24110)
-	else:
-		MoviesButton.SetFlags(IE_GUI_BUTTON_NO_IMAGE | IE_GUI_BUTTON_NO_TEXT, OP_SET)
-		MoviesButton.SetStatus(IE_GUI_BUTTON_DISABLED)
```

Comment:
Don't you now end up with a textless button that you can still click?

Reply:
You are right, I cleaned it up after too much faith in an empty `SetText`.